### PR TITLE
docs: introduce docs for page object models

### PR DESCRIPTION
**Pull request by arjunattam**
* Added docs for page object models, since we've seen interest on it from users.
* Also added a new section to docs called "tutorials" to complement "guides". While guides focus on specific actions (like input or console logs), tutorials will cover use-cases/scenarios. Name suggestions welcome!

### CODE REVIEW

**Review by mxschmitt — APPROVED** [review]

File: docs/pom.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+## Implementation
+Page object models wrap over a Playwright [`page`](./api.md#class-page).
+
+```js
+// in models/Search.js
+class SearchPage {
+  constructor(page) {
+    this.page = page;
+  }
+  async navigate() {
```
Comment (dgozman) [inline docs/pom.md:30]: `async goto()`

File: docs/pom.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+    await this.page.goto('https://bing.com');
+  }
+  async fill(text) {
```
Comment (dgozman) [inline docs/pom.md:33]:
```
async search(text) {
  await this.page.fill('[aria-label="Enter your search term"]', text);
  await this.page.keyboard.press('Enter');
}
```

File: docs/pom.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    await this.page.fill('[aria-label="Enter your search term"]', text);
+  }
+}
+module.exports = { SearchPage };
+```
+
+Page objects can then be used inside a test.
+
+```js
+// in search.spec.js
```
Comment (dgozman) [inline docs/pom.md:43]: This comment hints that we see a test file, but content does not. Maybe just a short snippet for now, without requires/launch?

**Review by dgozman — APPROVED** [review]

File: docs/pom.md (hunk 3 above)
Comment (arjun27) [reply docs/pom.md:43]: Sounds good. Thanks for the review!

**Pull request merged**